### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1247

### Change description ###
Dockerfile is required for the Jenkins pipeline. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.2
+
+COPY lib/AI-Agent.xml /opt/app/
+COPY build/libs/bulk-scan-end-to-end-tests.jar /opt/app/
+
+CMD [ "bulk-scan-end-to-end-tests.jar" ]
```
Comment (nitinprabhu) [inline Dockerfile:6]: do we need this ?

Comment (nitinprabhu) [inline Dockerfile:6]: do we need this ?

Comment (aliveni) [reply Dockerfile:6]: we don't need this but pipeline mandates to have Dockerfile

Comment (nitinprabhu) [reply Dockerfile:6]: I meant those 3 lines

File: charts/bulk-scan-end-to-end-tests/requirements.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+dependencies:
+  - name: java
+    version: ~2.16.1
```
Comment (doncem) [inline charts/bulk-scan-end-to-end-tests/requirements.yaml:3]: it's 2.18 now ;)

**Pull request merged**